### PR TITLE
Adds 3 new emotes to the game

### PR DESCRIPTION
**Pull request by etherware-novice**
# About The Pull Request
Adds a *thumbsup, *thumbsdown, and *whistle emote because i have to manual emote them otherwise

## Why It's Good For The Game
Adds more opprotunity for rp, flashing thumbs up or down is useful for replying to people, and whistle can be used in like awkward situations!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
add: Added thumbsup, thumbsdown, and whistle emote
soundadd: its supposed to play midi notes when using whistle, availability may vary
/:cl:

### CODE REVIEW

**Comment by etherware-novice** [comment]
NOOOOOOOO WHYYY CATGLOVES.DMI IS BACK WITH A VENGANCE

File: monkestation/code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	key = "thumbsdown"
+	key_third_person = "thumbsdown"
+	message = "flashes a thumbs down"
+	message_robot = "makes a crude thumbs down with their 'hands'"
+	message_AI = "flashes a quick hologram of a thumbs down"
+	message_ipc = "flashes a thumbs down icon"
+	message_simple = "attempts a thumbs down"
+	message_param = "flashes a thumbs down at %t"
+
+/datum/emote/living/whistles
```
Comment (nednaZ) [inline monkestation/code/modules/mob/living/emote.dm:21]: This is "whistles" while the get_sound is "whistle"

Comment (etherware-novice) [reply monkestation/code/modules/mob/living/emote.dm:21]: ohh thats why it didnt work

File: monkestation/code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+/datum/emote/living/whistle
+	key="whistle"
+	key_third_person="whistle"
+	message = "whistles a few notes"
+	message_robot = "whistles a few synthesized notes"
+	message_AI = "whistles a synthesized song"
+	message_ipc = "whistles a few synthesized notes"
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/living/whistle/get_sound(mob/living/user)
```
Comment (nednaZ) [inline monkestation/code/modules/mob/living/emote.dm:30]: The sound doesn't play when you whistle

**Review by nednaZ — CHANGES_REQUESTED** [review]
Looks fine outside the sound

File: monkestation/code/modules/mob/living/emote.dm (hunk 2 above)
Comment (etherware-novice) [reply monkestation/code/modules/mob/living/emote.dm:30]: strange...

**Comment by nednaZ** [comment]
Request another review once you have it working and tested yourself.

**Pull request closed without merging**